### PR TITLE
chore(build): rename proxy binary to `radicle-proxy`

### LINKED ISSUE #1441: Prefix proxy process to make it clear it belongs to radicle
Currently when debugging, esp. in scenarios where the proxy process is lingering, it's not always easy to spot that it belongs to radicle. A simple `radilce-` prefix would help to aid with that.

Ref. #1440

### PR DESCRIPTION
**Pull request by geigerzaehler**
Fixes #1441

### CODE REVIEW

**Comment by rudolfs** [comment]
```
work/radicle-upstream » yarn dist
yarn run v1.22.5
$ yarn dist:clean && yarn rollup:build && yarn proxy:build:release && electron-builder --publish never
error Command "dist:clean" not found.
```

File: package.json [hunk 1]
```diff
@@ -26,15 +26,15 @@
     },
     "extraResources": [
       {
-        "from": "dist/proxy",
-        "to": "proxy"
+        "from": "proxy/target/release/radicle-proxy",
```
Comment (rudolfs) [inline package.json:29]: Not sure this will work, but we can try it out.

Comment (geigerzaehler) [reply package.json:29]: I tried it and it works

**Comment by geigerzaehler** [comment]
I’ve fixed the issue with the `dist` command. (My fat finger deleted an additional line).

**Review by xla — APPROVED** [review]
☝️ 🌵 🌙 🌪

**Review by rudolfs — APPROVED** [review]

**Comment by geigerzaehler** [comment]
Merged in https://github.com/radicle-dev/radicle-upstream/commit/7c616fc05533faf898bfce777a4ea2ac436c671a

**Pull request closed without merging**